src/components/NavigationBarBrand/NavigationBarBrand.component.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Styled from './NavigationBarBrand.styled';
+
+function NavigationBarBrand() {
+  return (
+    <Styled.Container className="navbar-brand" href="/" data-testid="NavigationBarBrand">
+      <Styled.Image
+        src="./logo120.png"
+        alt=""
```

Comment:
We might want to specify an alt attribute for Images, we can also get by alt text later in tests if you cant to test.